site/docs/v1/tech/guides/breached-password-detection.adoc

```diff
@@ -32,7 +32,12 @@ Conceptually, FusionAuth uses a pull model rather than a push model. Passwords a
 
 The first step to enabling breached password detection is to have a valid license key. Please visit link:/pricing/[our pricing page] to buy a license.
 
-The next step is to activate the license. You'll need to ensure that your FusionAuth instance has outbound network access. To activate, simply follow the steps outlined in the link:/docs/v1/tech/reactor/[Reactor documentation].
+The next step is to activate the license. You'll need to ensure that your FusionAuth instance has outbound network access. To activate, simply follow the steps outlined in the link:/docs/v1/tech/reactor/[Reactor documentation]. 
+
+[NOTE]
+====
+After you activate your license, there will be a delay before the breached password dataset is available and the feature is working. This delay is typically minutes and at most an hour.
```

Comment:
Assuming this is just a guess, but it should never take an hour. Minutes at the most. I'm ok with leaving this as is - just to set expectations if that was your intent.

Reply:
I saw the weird behavior for a fair while, ~25 minutes on demo. I thought it'd be good to set expectations high.